<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="category">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="amount">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2025</year>
      <month>3</month>
      <day>24</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="submitEntry">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="results">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>1241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadEntries">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Load All Entries</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterByDate">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>30</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="totalButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Totals </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
